sbi/inference/abc/smcabc.py

```diff
@@ -246,13 +257,38 @@ def sass_simulator(theta):
                 log_weights=all_log_weights[-1],
                 lra_with_weights=lra_with_weights,
             )
-            posterior = Empirical(adjusted_particels, log_weights=adjusted_weights)
+            final_particles = adjusted_particels
```

Comment:
Typo: particels -> partciles (source is return of `self.run_lra_update_weights` above)